pyproject.toml

```diff
@@ -54,6 +54,9 @@ optional = [
     "botorch",
```

Comment:
Should we add version constraint to this line also?

Reply:
Thank you. Updated.